integration/inspektor-gadget/run_snapshot_process_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/run/types"
+
+	. "github.com/inspektor-gadget/inspektor-gadget/integration"
+)
+
+func TestRunSnapshotProcess(t *testing.T) {
+	ns := GenerateTestNamespaceName("test-run-snapshot-process")
+
+	t.Parallel()
+
+	// TODO: Handle it once we support getting container image name from docker
+	isDockerRuntime := IsDockerRuntime(t)
+
+	commandsPreTest := []*Command{
+		CreateTestNamespaceCommand(ns),
+		BusyboxPodCommand(ns, "nc -l -p 9090"),
```

Comment:
Any thing that is run as a **process** and lasts long enough to be detected by the gadget should work.
`sleep inf` doesn't work on this case because it's a shell builtin in busybox:

```bash
$ docker run --rm -it docker.io/library/busybox:latest sh -c "type sleep"
sleep is a shell builtin
```